File: Math/SWEA 8016/src.py
T = int(input())        # 테스트 케이스의 수

for j in range(1, T + 1):
    i = int(input())    
    if i == 1:
        print(f"#{j} {1} {1}")
    else:
        left = 2 * (i - 1)**2 + 1
        right = 2 * (i)**2 - 1
        print(f"#{j} {left} {right}")

File: Math/SWEA 8016/src.cpp
#include <iostream>

using namespace std;

int main() {
  unsigned long long int test, i;
  cin >> test;

  for (int j = 1; j <= test; j++){
    cin >> i;
    if (i == 1){
      cout << '#' << j << " " <<  1 << " " << 1 << endl;
    }
    else{
      unsigned long long int left = 2 * (i - 1) * (i - 1) + 1;
      unsigned long long int right = 2 * i * i - 1;
      cout << '#' << j << " " <<  left << " " << right << endl;
    }
  }
  
  return 0;
}
